pipelines/templates/copy-and-publish-powershell.yml

```diff
@@ -58,19 +58,19 @@ steps:
      ${{ parameters.templatesPath }}\**\frontdoor.json
      ${{ parameters.templatesPath }}\**\keyvault.json
      ${{ parameters.templatesPath }}\**\storageaccount.json
-    TargetFolder: '$(build.artifactstagingdirectory)\Winget.Powershell.Source\Library\ARMTemplate'
+    TargetFolder: '$(build.artifactstagingdirectory)\Winget.PowerShell.Source\Library\ARMTemplate'
     OverWrite: true
     flattenFolders: true
 
 - task: CopyFiles@2
   displayName: 'Copy Files: azure function'
   inputs:
     SourceFolder: '$(Build.ArtifactStagingDirectory)\WinGet.RestSource.Functions'
-    TargetFolder: '$(build.artifactstagingdirectory)\Winget.Powershell.Source\Library\RestAPI'
+    TargetFolder: '$(build.artifactstagingdirectory)\Winget.PowerShell.Source\Library\RestAPI'
     OverWrite: true
 
 - task: PublishBuildArtifacts@1
-  displayName: 'Publish Artifact: WinGet.RestSource-Winget.Powershell.Source'
+  displayName: 'Publish Artifact: WinGet.RestSource-Winget.PowerShell.Source'
   inputs:
-    PathtoPublish: '$(build.artifactstagingdirectory)\Winget.Powershell.Source'
+    PathtoPublish: '$(build.artifactstagingdirectory)\Winget.PowerShell.Source'
     ArtifactName: 'WinGet.RestSource-Winget.Powershell.Source'

```

Comment:
Fix PowerShell casing here too?

Reply:
Ah yes. the artifact name is also important.